core/core_bind.h

```diff
@@ -248,7 +248,9 @@ class OS : public Object {
 		SYSTEM_DIR_DCIM,
 		SYSTEM_DIR_DOCUMENTS,
 		SYSTEM_DIR_DOWNLOADS,
+#ifndef MOVIE_WRITER_DISABLED
 		SYSTEM_DIR_MOVIES,
+#endif // MOVIE_WRITER_DISABLED
```

Comment:
That's unrelated to movie writer, it's just a system path like `~/Videos` on Linux.